### DISCUSSION TITLE
Add options to make Windows 10 look like Windows 11

### DISCUSSION THREAD

**Discussion by Rellawing** [valinet/ExplorerPatcher#2040] [Ideas]
Would you mind adding more options that would transform Windows 10 to Windows 11? The addition of true rounded corners on the same elements that Windows 11 users enjoy, the same clock configuration, adjustments to the action center icon. Positioning of the weather widget you add to the left of the taskbar (the left positioning puts it to the right of the start button, which doesn't look good, and the widget won't even show there, anyway for some reason).

If you could add these things, it would make transforming windows 10 into 11 far more complete, of course minus the settings app appearances and such.

**Comment by pyrates999** [comment]
If you mean the true rounded corners that windows 7 had, there's a story behind that.  It was some gpu that Microsoft wanted to support for windows 8, but it did not support the aeroglass and rounded corners that was in windows 7.  So the programmers were hastily told to just rip out all of that code for windows 8.  And then we had square corners, and I've hated it ever since.

I was finally happy when windows 11 was getting the rounded corners back, but could not stand the other UI changes like the start menu and the changing of the right click menu.  But windows 10 doesn't have any code for rounded corners.  So I don't think that part of this is possible.

And as for transforming windows 10 to 11, they can just upgrade to windows 11 if they want that.  The code for those features has to exist in windows 10 in order for EP to be able to access it.

**Reply by Rellawing** [reply]
I love how soothing the Windows 11 rounded corners feel. I had the opportunity to use it for a short time until the android subsystem triggered a catastrophic failure and forced me to install a fresh copy of Windows 10 again.

I can't upgrade to Windows 11, since there are some severe landmines hidden in it. (Not to mention the errors that falsely triggered an 'incompatible processor' error. My computer's processor is technically not supported, but ran Windows 11 beautifully. It's just ironic that I fell afoul of that too.
